omero/sysadmins/unix/server-centos6.txt

```diff
@@ -1,36 +1,47 @@
-Installing Python 2.7 and Ice 3.5 on CentOS 6
-=============================================
+OMERO.server installation on CentOS 6 with Python 2.7 and Ice 3.5
+=================================================================
+
+This installation walkthrough should be read in conjunction with
+:doc:`server-installation` and :doc:`install-web`.
 
 Running OMERO on CentOS 6 has a number of special requirements which
-deviate from the standard installation instructions.  The instructions
-below will set up Python 2.7 and Ice 3.5 on CentOS 6.
+deviate from the standard installation instructions. The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6. We tested the
+installation with Python 2.7 from `IUS <https://ius.io/>`_ and used
+a `virtual environment <https://virtualenv.readthedocs.org/en/latest/>`_
+to install the various dependencies required to install an OMERO.server.
+It is also possible to use SCL Python (for example 
+:download:`install_centos6_py27_nginx.sh <walkthrough/install_centos6_py27_nginx.sh>`) but such solution could have
+potential side effects.
 
 Python 2.7
 ----------
 
-CentOS 6 provides Python 2.6.  However, OMERO.web requires Python 2.7
-in order to use django 1.8.  While django 1.6 may be used with Python
-2.6, this version of django no longer has security support.  In
+CentOS 6 provides Python 2.6. However, OMERO.web requires Python 2.7
+in order to use `Django 1.8`. While `Django 1.6` may be used with Python
+2.6, this version of Django no longer has security support.  In
 consequence, it is necessary to upgrade to Python 2.7 in order to
-obtain django security updates, which are required for a production
+obtain Django security updates, which are required for a production
 deployment.
 
-As root, run::
+**The following steps are run as root.**
+
+
+Install:
```

Comment:
should be `::` here, we've got purple highlighted indenting for the command rather than a code box